style.css

```diff
@@ -132,6 +132,7 @@ li {
 }
 
 
```

Comment:
random comment (but) maybe you can put  this in your CSS to allow smooth scrolling.

* {
     scroll-behavior: smooth;
}